### PR TITLE
CDPS-335 neurodiversity section

### PR DESCRIPTION
**Pull request by jamesgrant-moj**

### CODE REVIEW

File: server/views/partials/personalPage/neurodiversity.njk [hunk 1]
```diff
@@ -1,16 +1,87 @@
...
+          To check for neurodiversity information for someone in another prison, contact the education team in that
+          prison.
+        </p>
+      {% elseif learnerNeurodivergence.length %}
+        <p class="govuk-body" data-qa="neurodivergence-exists">
+          Information from the local education team - not a medical diagnosis.
+        </p>
+
+        {# Support needed #}
+        {% if learnerNeurodivergence[0].neurodivergenceSupport.length %}
```
Comment (davidmiddletonmoj) [inline server/views/partials/personalPage/neurodiversity.njk:22]: Not sure what the actual data looks like, but should this always be hard-coded to the first entry in the array?

Can there be multiple entries? And if so, do they all need to be displayed, or only (latest?) one? What order does the array return entries?

**Review by davidmiddletonmoj — COMMENTED** [review]
Looks good, but one comment about the array
Also, e2e tests appear broken in pipeline

File: server/views/partials/personalPage/neurodiversity.njk (hunk 1 above)
Comment (jamesgrant-moj) [reply server/views/partials/personalPage/neurodiversity.njk:22]: Thanks for taking a look, yes all the curious API responses are arrays which contain single objects.
[https://editor.swagger.io/?url=https://raw.githubusercontent.com/ministryofjustice/curious-API/main/curious-api-specification.yaml](url)

**Review by davidmiddletonmoj — APPROVED** [review]
LGTM

**Pull request merged**